common-lib/src/main/scala/com/gu/mediaservice/lib/logging/MarkerAugmentation.scala

```diff
@@ -0,0 +1,25 @@
+package com.gu.mediaservice.lib.logging
+
+import net.logstash.logback.marker.Markers.appendEntries
+import org.slf4j.Marker
+import play.api.{Logger, MarkerContext}
+import scala.collection.JavaConverters._
+
+
+object MarkerAugmentation {
+  /** *
+    *
+    * @param markerContext
+    * @param augmentation : more details!
+    * @return a new markercontext!
+    *         it wouldn't be java if it could be cloned
+    */
+  def augmentMarkerContext(markerContext: MarkerContext, augmentation: Map[String, Any]): MarkerContext = {
+    val newMarker = appendEntries(augmentation.asJava)
+    markerContext.marker.foreach(newMarker.add)
```

Comment:
What's the value of `markerContext.marker` here?

Reply:
`markerContext.marker` is an `Option[Marker]`